organizing_hub/templates/partials/organizing_hub_header.html

```diff
@@ -26,8 +26,12 @@
             {% if not static_content and request.user.is_authenticated %}
             <div class="col-md-6">
               <div class="uppercase text-right">
+                {% local_group as group %}
```

Comment:
Are we sure `local_group` is always available? This is a header used on every page. I'm concerned that this is not always in the model/context? Or might even return the wrong object in some cases? We specifically want to make sure this is the local_group associated with the logged in user. Are we confirming this somehow? For example the `group_link` does not use `local_group` form context, it uses a template tag.

Reply:
This is a template tag that simply calls `find_local_group_by_user().` `{% group_link %}`, which was here before, essentially calls the same thing.

Reply:
ok thanks for explaining, didn't realize this was a template tag. we should give it a better name. also would be nice to optimize and not do the same db lookup twice in this case. like either get the group once and pass it into the group link tag, or just check if the group link is none. not critical, but seems like not good performance to lookup same query twice in a row here.